inter-page/marketing-bot/app.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ * Starter Project for Marketing bot of the Messenger Platform Inter-bot Tutorial
+ *
+ * Use this project as the starting point for following the tutorial.
+ *
+ * TODO: Waiting for link
+ *
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    ACCESS_TOKEN = process.env.ACCESS_TOKEN,
+    app = express().use(body_parser.json()); // creates express http server
+
+// Sets server port and logs message on success
+app.listen(process.env.PORT || 5000, () => console.log('webhook is listening'));
+
+app.post('/webhook', (req, res) => {
+    // Parse the request body from the POST
+    let body = req.body;
+
+    // Check the webhook event is from a Page subscription
+    if (body.object === 'page') {
+        body.entry.forEach(function (entry) {
+            if (entry.messaging) {
+                // Gets the body of the webhook event
+                let webhook_event = entry.messaging[0];
+
+                // Get the sender PSID
+                let sender_psid = webhook_event.sender.id;
+                console.log('Sender PSID: ' + sender_psid);
+
+                // Check if the event is a message or postback and
+                // pass the event to the appropriate handler function
+                if (webhook_event.message) {
+                    handleMessage(sender_psid, webhook_event.message);
+                }
+            } else if (entry.changes) {
+                processComments(entry.changes[0].value);
+            }
+        });
+
+        // Return a '200 OK' response to all events
+        res.status(200).send('EVENT_RECEIVED');
+
+    } else {
+        // Return a '404 Not Found' if event is not from a page subscription
+        res.sendStatus(404);
+    }
+
+});
+
+function processComments(comment) {
```

Comment:
add description of stub

Reply:
@ChrisChinchilla This still needs update

Reply:
@ChrisChinchilla Sorry, copied 'stub' from the starter. Still needs a description tho.